### DISCUSSION TITLE
posts/custom-matplotlib-style/

### DISCUSSION THREAD

**Discussion by giscus** [anyuzx/Prongs#50] [Announcements]
# posts/custom-matplotlib-style/

A note on customize matplotlib style | Guang Shi's personal site. Research, blog and photography

https://www.shisguang.com/posts/custom-matplotlib-style/

**Comment by anyuzx** [comment]
Test use giscus for comment. Support KaTex $\sqrt{3x-1}+(1+x)^2$

**Reply by anyuzx** [reply]
How do I delete this?
